build-system/tasks/runtime-test/helpers-test-reporting.js

```diff
@@ -0,0 +1,86 @@
+/**
+ * Copyright 2019 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+'use strict';
+
+const fetch = require('node-fetch');
+const fs = require('fs').promises;
+const log = require('fancy-log');
+const {
+  travisBuildNumber,
+  travisJobNumber,
+  travisCommitSha,
+} = require('../../common/travis');
+
+const JSON_REPORT_TEST_TYPES = new Set(['unit', 'integration']);
```

Comment:
`unit` and `integration` isn't enough to differentiate between the test types that Travis runs:
- integration for compiled code (dist tests job)
- integration for uncompiled code (local tests job)
- unit for uncompiled code (local tests job)
- integration for experimental code (experiment A/B/C tests jobs)

Unless, is the plan to job and build info as well?

Reply:
The plan is to use job and build info as well -- for example, the database would have one entry looking like
`{job_num: 12.34, test_type: 'unit'}` and another entry looking like
`{job_num: 12.34, test_type: 'integration'}`.

There is something odd about having job_nums not be identifiers, though, any suggestions?

Reply:
Job nums won't be unique for re-runs of jobs either. That should be fine :)(